### PR TITLE
Fix datepicker

### PR DESCRIPTION
**Pull request by chipit24**
This PR fixes an issue with the datepicker not displaying the set date value on its input.

### CODE REVIEW

**Comment by JCook21** [comment]
@chipit24 looks great. Could you add some comments so that @sanjacornelius and @caleeli know why these changes were made? They weren't in the room with us and don't have any of the context.

File: src/components/FormDatePicker.vue [hunk 1]
```diff
@@ -28,7 +29,7 @@
 
   const uniqIdsMixin = createUniqIdsMixin();
 
-  const datetimeStdFormat = 'YYYY-MM-DDThh:mm:ssZZ';
+  const datetimeStdFormat = 'YYYY-MM-DDTHH:mm:ssZZ';
```
Comment (chipit24) [inline src/components/FormDatePicker.vue]: The capital `H` formats the value in 24-hour time, preserving the difference between AM/PM, and allowing the modeler to correctly interpret the date (as the modeler expects a (24-hour) ISO format date string.

File: src/components/FormDatePicker.vue [hunk 2]
```diff
@@ -3,7 +3,8 @@
     <label v-uni-for="name">{{label}}</label>
     <date-picker
       :config="config"
-      v-model="date"
+      :value="moment(date)"
```
Comment (chipit24) [inline src/components/FormDatePicker.vue]: This appears to be a bug with the date picker. Passing in a moment object allows the date picker to correctly display the date in its input.

**Comment by chipit24** [comment]
Added some comments describing the issue.

**Pull request merged**